### DISCUSSION TITLE
Auto-editor export premiere file not editting correctly

### DISCUSSION THREAD

**Discussion by xivernn** [WyattBlue/auto-editor#404] [1. Help]
Hi! I have run into an issue where the file I am trying to edit isn't edited correctly at the silences whenever exporting as premiere pro. When I open the xml file within premiere pro, it will have edited and cut during a vocal segment while leaving in the silences at other sections. Causing the audio to be incoherent. This doesn't happen when exporting the file as an mp4 file. However, I would like the video to be editable/nondestructive in case silence is needed or a visual is present. 

Some information about the file and commands I am using:
- the file type is mp4 with video and audio
- there are two audio tracks within the file, one commentary while the other is game sounds
- the audio is rather low overall, however, there are clear peaks and valleys (it hovers around -25db at the lowest point while talking)
- the most common command I have run is **auto-editor "2023-08-19 15-36-40.mp4" --margin 0.8sec --export premiere**
- I have run variants such as **auto-editor "2023-08-19 15-36-40.mp4" --edit "(or audio:stream=0 audio:threshold=-25dB)" --export premiere** with zero results or difference in the cuts

The only time I have received a difference in the xml file is when I combined the tracks and increased overall volume. It was there that I received the results that I was looking for. However, if I am able to find a solution without having to combine the two tracks, that would be preferred. Thank you so much for any help you are able to provide!

**Comment by LonelyFarts** [comment]
Hey Wyatt, I also have this issue and I want to say I'm available as a test case if you'll have me. The results when exporting to premiere pro are often bafflingly bad. Like, sentences/words cut off followed by four 10 second clips in a row of absolute silence. It's a dumb bug somewhere in your code and I know it's a dumb bug because the results are so polarized. One video will be edited absolutely perfect and then the next will be complete dog shit. Despite video 1 and video 2 being the same video that my phone splits into 4gb chunks, it's the same camera in the same position in the same house with the same idiot rambling. Let me know man. I believe in this app and I'd love to help track this bug down in any way I can.

Thanks,
Nathan Zurowski

**Comment by dr04e606** [comment]
This odd behavior might be due to variable frame rate (VFR), which can be fixed by converting your source footage to constant frame rate (CFR) with HandBrake.
